src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/CompareUtil.java

```diff
@@ -39,9 +39,10 @@
      * Tells if the given pattern matches the string according to the implemented comparer/algorithm.
      * @param pattern the pattern to use.
      * @param str the string to match on.
+     * @param envVars the environment variables exisiting on the jenkins host.
      * @return true if the string matches the pattern.
      */
-    boolean matches(String pattern, String str);
+    boolean matches(String pattern, String str, EnvVars envVars);
```

Comment:
I didn't realise that this code might be used by other plugins, I guess there isn't really a way to figure this out?